rllib/agents/dreamer/dreamer.py

```diff
@@ -211,13 +218,34 @@ def validate_config(self, config: TrainerConfigDict) -> None:
             raise ValueError("Distributed Dreamer not supported yet!")
         if config["clip_actions"]:
             raise ValueError("Clipping is done inherently via policy tanh!")
+        if config["dreamer_train_iters"] <= 0:
+            raise ValueError(
+                "`dreamer_train_iters` must be a positive integer. "
+                f"Received {config['dreamer_train_iters']} instead."
+            )
         if config["action_repeat"] > 1:
             config["horizon"] = config["horizon"] / config["action_repeat"]
 
     @override(Trainer)
     def get_default_policy_class(self, config: TrainerConfigDict):
         return DreamerTorchPolicy
 
+    @override(Trainer)
+    def setup(self, config: PartialTrainerConfigDict):
+        super().setup(config)
+        # `training_iteration` implementation: Setup buffer in `setup`, not
+        # in `execution_plan` (deprecated).
+        if self.config["_disable_execution_plan_api"] is True:
```

Comment:
Nice! I think at some point, we should also move this into the new replay buffer (config) API. @ArturNiederfahrenhorst can help with this (in a separate PR after we merged this one here).

Reply:
+1